.buildkite/dagster-buildkite/dagster_buildkite/cli.py

```diff
@@ -19,20 +21,20 @@ def dagster():
     if not dagit_only:
         all_steps += dagster_steps()
 
-        all_steps += [wait_step()]
-
-        if DO_COVERAGE:
-            all_steps += [coverage_step()]
-
-        # Trigger builds of the internal pipeline for builds on master
+        # Trigger builds of the internal pipeline.
         all_steps += [
             trigger_step(
                 pipeline="internal",
-                async_step=True,
+                async_step=os.getenv("BUILDKITE_BRANCH", "master") == "master",
                 if_condition="build.branch=='master' && build.creator.email =~ /elementl.com$$/",
```

Comment:
Don't we need to remove this condition? Otherwise I think it'll still only trigger internal on `master` builds.